cmake/Corrosion.cmake

```diff
@@ -171,8 +171,24 @@ function(_add_cargo_build)
 
     set(target_artifact_dir "${_CORROSION_RUST_CARGO_TARGET}")
 
+    if(COR_ALL_FEATURES)
+        set(all_features_arg --all-features)
+    endif()
+    if(COR_NO_DEFAULT_FEATURES)
+        set(no_default_features_arg --no-default-features)
+    endif()
+
     if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.19.0)
         set(build_env_variable_genex "$<GENEX_EVAL:$<TARGET_PROPERTY:${target_name},CORROSION_ENVIRONMENT_VARIABLES>>")
+        
+        set(features_target_property "$<GENEX_EVAL:$<TARGET_PROPERTY:${target_name},CORROSION_FEATURES>>")
+        set(features_genex "$<$<BOOL:${features_target_property}>:--features=$<JOIN:${features_target_property},$<COMMA>>>")
+
+        # target property overrides corrosion_import_crate argument
+        set(all_features_target_property "$<GENEX_EVAL:$<TARGET_PROPERTY:${target_name},CORROSION_ALL_FEATURES>>")
+        set(all_features_arg "$<IF:$<BOOL:${all_features_target_property}>,${all_features_target_property},${all_features_arg}>")
```

Comment:
This override isn’t working, it’s missing the Boolean to argument transformation.

Reply:
This is fixed now